ui-tests/src/test/java/org/openmrs/reference/SearchActiveVisitsTest.java

```diff
@@ -31,7 +31,7 @@ public void setup() {
         new TestData.TestVisit(patient.uuid, TestData.getAVisitType(), getLocationUuid(homePage)).create();
     }
 
-    @Ignore
+    @Test
```

Comment:
Add `@Category(BuildTests.class)` too

Reply:
Thanks @kaweesi , this is already been added here thanks, it passess on both drivers